qbft/spectest/tests/timeout/test.go

```diff
@@ -55,5 +57,5 @@ func (test *SpecTest) Run(t *testing.T) {
 
 	postRoot, err := test.Pre.State.GetRoot()
 	require.NoError(t, err)
-	require.EqualValuesf(t, test.PostRoot, hex.EncodeToString(postRoot), "post root not valid")
+	require.EqualValuesf(t, test.PostRoot, hex.EncodeToString(postRoot[:]), "post root not valid")
```

Comment:
would it be helpful to print the diff here as well?

Reply:
not related to this scope. Will be done only after implementing state comparison to those tests